src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -84,22 +92,22 @@
     /** Period literal. */
     private static final String PERIOD = ".";
 
-    /**
-     * Stores allowed values in document for inherit doc literal.
-     */
-    private static final Set<Integer> SKIP_TOKENS = new HashSet<>(
-        Arrays.asList(JavadocTokenTypes.NEWLINE,
-                       JavadocTokenTypes.LEADING_ASTERISK,
-                       JavadocTokenTypes.EOF)
+    /** Inherit doc literal. */
+    private static final String INHERIT_DOC = "{@inheritDoc}";
```

Comment:
it is already there - https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java#L639

you do not need your own definition of it.